samples/snippets/src/main/java/com/google/cloud/translate/automl/ModelApi.java

```diff
@@ -47,48 +41,6 @@
  */
 public class ModelApi {
 
-  // [START automl_translate_create_model]
```

Comment:
We need to keep one copy of automl_translate_create_model around.

Reply:
https://github.com/googleapis/java-automl/blob/fb9dad14aac1d684ea136ccca158798545ea26f3/samples/snippets/src/main/java/beta/automl/TablesCreateModel.java#L19